conf/lookup/BCD-Port.csv

```diff
@@ -11479,7 +11478,7 @@ PortId,CountryCode,PortType,PortCode,KeyWords,Name,Name-cy,Priority,Region,Locat
 32563,UK,3,MRY,,Maryport Seaport - MRY,Maryport Seaport - MRY,,1,NULL,1,
 32578,UK,3,MEB,,Menai Bridge Seaport - MEB,Menai Bridge Seaport - MEB,,1,NULL,1,
 32584,UK,3,MTH,,Methil Seaport - MTH,Methil Seaport - MTH,,1,NULL,1,

```

Comment:
Also part of 00031,UK,3,GRG,Braefoot Bay Burntisland Inverkeithing Methil,Grangemouth - GRG,Grangemouth - GRG,,1,NULL,0,


Just keep both